### DISCUSSION TITLE
wildcards is not observed in the config path.

### DISCUSSION THREAD

**Discussion by StefanSa** [vectordotdev/vector#12088] [Q&A] [answered]
Hi there,
I start vector in the console this way:
`vector -c /etc/vector/vector.toml -c /etc/vector/modules/*/*.toml`
All `toml` files were loaded.

The same configuration in `systemd` does not work.
```
/etc/vector/modules
                 └───sophos-xg/*.toml
                 └───telegraf/*.toml
```
Here only the first found `toml` is loaded, in the case in the folder `sophos-xg`.
Any solution to this problem ?

**Comment by jszwedko** [comment, marked as answer]
Hi @StefanSa ,

I don't think SystemD evaluates wildcards. See https://askubuntu.com/a/1162995

**Reply by StefanSa** [reply]
As always, thanks for your help.
